jbake-master: "jbake -b" shows java.util.NoSuchElementException: 'index.paginate'...

* jbake-2.5.0: No error/warning
* jbake-master: Displays an error/warning

```
$ jbake -b
...
07:22:19.167 INFO  org.jbake.app.Asset - Copying [./assets/js/i2ui-1.0.0.js]...done!
07:22:19.168 INFO  org.jbake.app.Asset - Copying [./assets/js/jquery-1.11.1.min.js]...done!
07:22:19.170 INFO  org.jbake.app.Asset - Copying [./assets/js/prettify.js]...done!
07:22:19.171 INFO  org.jbake.app.Oven - Baking finished!
07:22:19.173 INFO  org.jbake.app.Oven - Baked 188 items in 56361ms
07:22:19.173 ERROR org.jbake.app.Oven - Failed to bake 1 item(s)!
07:22:19.215 INFO  c.o.orient.core.Orient - - storage: cache...
07:22:19.247 INFO  c.o.orient.core.Orient - OrientDB Engine shutdown complete
JBake failed with 1 errors:
1. java.util.NoSuchElementException: 'index.paginate' doesn't map to an existing object

org.jbake.app.JBakeException: JBake failed with 1 errors:
1. java.util.NoSuchElementException: 'index.paginate' doesn't map to an existing object

	at org.jbake.launcher.Baker.bake(Baker.java:32)
	at org.jbake.launcher.Main.run(Main.java:95)
	at org.jbake.launcher.Main.run(Main.java:81)
	at org.jbake.launcher.Main.main(Main.java:34)
Caused by: org.jbake.template.RenderingException: java.util.NoSuchElementException: 'index.paginate' doesn't map to an existing object
	at org.jbake.render.IndexRenderer.render(IndexRenderer.java:24)
	at org.jbake.app.Oven.bake(Oven.java:151)
	at org.jbake.launcher.Baker.bake(Baker.java:20)
	... 3 more
Caused by: java.util.NoSuchElementException: 'index.paginate' doesn't map to an existing object
	at org.apache.commons.configuration.AbstractConfiguration.getBoolean(AbstractConfiguration.java:644)
	at org.jbake.render.IndexRenderer.render(IndexRenderer.java:17)
	... 5 more
```

--- a/src/main/java/org/jbake/render/IndexRenderer.java
+++ b/src/main/java/org/jbake/render/IndexRenderer.java
@@ -1,6 +1,7 @@
 package org.jbake.render;
 
 import org.apache.commons.configuration.CompositeConfiguration;
+import org.jbake.app.ConfigUtil;
 import org.jbake.app.ConfigUtil.Keys;
 import org.jbake.app.ContentStore;
 import org.jbake.app.Renderer;
@@ -14,7 +15,7 @@ public class IndexRenderer implements RenderingTool {
     public int render(Renderer renderer, ContentStore db, File destination, File templatesPath, CompositeConfiguration config) throws RenderingException {
         if (config.getBoolean(Keys.RENDER_INDEX)) {
             try {
-                if (config.getBoolean(Keys.PAGINATE_INDEX)) {
+                if (shouldPaginateIndex(config)) {
                     renderer.renderIndexPaging(config.getString(Keys.INDEX_FILE));
                 } else {
                     renderer.renderIndex(config.getString(Keys.INDEX_FILE));
@@ -26,6 +27,10 @@ public class IndexRenderer implements RenderingTool {
         } else {
             return 0;
         }
+    }
+
+    private boolean shouldPaginateIndex(CompositeConfiguration config) {
+        return config.containsKey(Keys.PAGINATE_INDEX) && config.getBoolean(Keys.PAGINATE_INDEX);
     }
 
 }

--- a/src/test/java/org/jbake/render/IndexRendererTest.java
+++ b/src/test/java/org/jbake/render/IndexRendererTest.java
@@ -95,6 +95,24 @@ public class IndexRendererTest {
         verify(mockRenderer, never()).renderIndex("random string");
     }
 
+
+    /**
+     * @see <a href="https://github.com/jbake-org/jbake/issues/332">Issue 332</a>
+     */
+    @Test
+    public void shouldFallbackToStandardIndexRenderingIfPropertyIsMissing() throws Exception {
+        IndexRenderer renderer = new IndexRenderer();
+
+        MockCompositeConfiguration compositeConfiguration = new MockCompositeConfiguration().withDefaultBoolean(true);
+        ContentStore contentStore = mock(ContentStore.class);
+        Renderer mockRenderer = mock(Renderer.class);
+
+        int renderResponse = renderer.render(mockRenderer, contentStore,
+                new File("fake"), new File("fake"), compositeConfiguration);
+
+        verify(mockRenderer, times(1)).renderIndex("random string");
+    }
+
 }
 
 

